app/cards/[id]/charts/constant.ts

```diff
@@ -0,0 +1,145 @@
+export const DEFAULT_HEIGHTforAGE = [
+    {
+      age: 0,
+      height_upper: 53,
```

Comment:
Usually, we use `camelCase` in JS/TS, unless some API returns data in a different case. This is more like a suggestion rather a problem to fix :)